Add surface index to nixie map to allow reliable use of nixies on non…

justarandomgeek opened the pull request:
…-default surfaces, for example with the Subsurface or NESTT mods.

justarandomgeek committed:
Add surface index to nixie map to allow reliable use of nixies on non-default surfaces, for example with the Subsurface or NESTT mods.

GopherAtl commented:
Hrm. Didn't notice this PR until after I'd merged the other, more recent one; there's a merge conflict, quite possibly just the version number update, but the computer I'm limited to atm is a dinky laptop that doesn't have either git or factorio installed, could I talk you into doing the merge on your end and updating the PR?

justarandomgeek commented:
No problem, I’ll do that later tonight!

Edit: upon further review, I see that we both changed the data structure, so it may take me a day or two to get the time to wrap my head around that and handle it properly!

justarandomgeek committed:
Faster refresh rate

justarandomgeek committed:
Merge remote-tracking branch 'remotes/GopherAtl/master'

# Conflicts:
#	control.lua

justarandomgeek committed:
Bump version again to allow clean data migration from the prior subsurface patch.

justarandomgeek commented:
This collection will cleanly handle updating from the earlier big-tubes-only builds, the big-and-small-builds, or previous multi-surface patch in existing savefiles. I had to use up multiple version numbers to do this, so I bumped it to 0.2.1. I tried doing this with 0.1.10 and 0.1.11, but the comparisons did not work correctly.

justarandomgeek committed:
Fix 0.1.9 migration file to apply cleanly to a save that has never has the mod installed.

justarandomgeek committed:
Updates for Factorio 0.13. Version Bump.

justarandomgeek committed:
Add both wires together

justarandomgeek committed:
Clean up gitignore and copy in my release script

justarandomgeek committed:
convert to unit_number indexed data

justarandomgeek committed:
fix for loading in a new save

justarandomgeek committed:
AlphaNixies and new sprites by GotLag. Color support on AlphaNixie device.

originalfoo committed:
control.lua cleanup and simplification

1. Defined globals for luacheck linter
2. Replaced unused loop vars with _
3. Removed whitespace at end of lines
4. Added lookup table `validEntityName` (truthy vals = num sprites)
5. Simplified onPlaceEntity()
6. Ensure local vars in displayValue()
7. Extraneous `local open=false` in onTickController() & onTickAlpha()
8. Extraneous tick check in onTick()
9. Simplify onRemoveEntity()
10. Extraneous `data` in script.on_init()

justarandomgeek committed:
Merge pull request #2 from aubergine10/master

control.lua cleanup and simplification

justarandomgeek committed:
Disable car sounds/headlights and remove car inventories.

justarandomgeek committed:
0.2.6

justarandomgeek committed:
Better coloring, using only a single layer of sprites.

justarandomgeek committed:
Tweaked Colors, follow .use_colors setting.

justarandomgeek committed:
Apply color to numeric nixies, and only apply color if "use colors" is set.

justarandomgeek committed:
0.2.7 - Improved colors

Color only applied when "use colors" is set.
Colors applied to numeric nixie groups.
Cleaner colors.

justarandomgeek committed:
Significantly improved performance. Numeric nixies now update every 5 ticks, and Alpha nixies update 10 units per tick, cycling around all units in the world.

justarandomgeek committed:
Update for Factorio 0.14

justarandomgeek committed:
Fixed Updating from very old versions - I now rebuild tables from scratch on *any* update, so it always works.
Fixed inverted "use colors" on Alpha Nixies.
Numeric Nixies now display 0 immediately on building.

justarandomgeek committed:
0.3.10 fork for Factorio 0.14
